rfd/0043-kubeaccess-multiparty.md

```diff
@@ -3,65 +3,70 @@ authors: Joel Wejdenstål (jwejdenstal@goteleport.com)
 state: implemented
 ---
 
-# RFD 43 - Shared sessions with observers for Kubernetes Access
+# RFD 43 - Moderated Sessions
 
 ## What
 
-Implement joint observer support for Kubernetes Access with support for configurable conditions similar to those of [RFD 26](https://github.com/gravitational/teleport/blob/2fd6a88800604342bfa6277060b056d8bf0cbfb2/rfd/0026-custom-approval-conditions.md).
-Also support defining conditions for required observers in order to initiate and maintain a session.
+Implement the ability to define RBAC policies with filters similar to those of [RFD 26](https://github.com/gravitational/teleport/blob/2fd6a88800604342bfa6277060b056d8bf0cbfb2/rfd/0026-custom-approval-conditions.md) that can be used to require others to be present in a session for it to be usable.
 
 ## Why
 
-Heavily regulated and security critical industries require that one or more observers with a certain role
-are present in Kubernetes Access sessions and viewing it live in order to guarantee that
-operator does not perform any mistakes or acts of malice.
+Heavily regulated and security-critical industries require that one or more participants with a certain role
+are present in SSH and Kubernetes sessions and viewing it live to guarantee that
+the operator does not perform any mistakes or acts of malice.
 
-Such observers need to have the power to terminate a session immediately should anything go wrong.
+Such participants need to have the power to terminate a session immediately should anything go wrong.
 
 To suit everyone this will need a more detailed configuration model based on rules
-that can be used to define observers, their powers and when and in what capacity they are required.
+that can be used to define observers, their powers, and when and in what capacity they are required.
 
 ## Details
 
-### Multiparty sessions
+### Terminology
 
-SSH sessions via TSH currently have rich support for sessions with multiple users at once.
+This RFD repeatedly refers to some specific nouns of which explanations can be found below:
+- Participant: Any user in a session regardless of mode, including the session initiator.
```

Comment:
@xacrimon thoughts on adding definition for `initiator`?

Reply:
Done